rfc_backmatter.markdown

```diff
@@ -63,6 +61,60 @@
     <author>
       <organization>The WebM Project</organization>
     </author>
-    <date month="November" year="2017" />
+    <date month="November" year="2017" day="28" />
```

Comment:
Why not `<date year="2017" month="November" day="28"/>`?

Reply:
No particular reason. In the end mmarkdown reformats it anyway.

Reply:
okay, thanks!